### ISSUE TITLE
Merge with jupyter-ros?

### ISSUE THREAD

**Issue by wolfv** [zmk5/jupyter-ros2#9]
Hi @zmk5 

I wanted to ask if you think we can merge this project back into jupyter-ros? I think it would be great to bundle our efforts. And I am also happy to give you commit rights to the original repository.
Do you think it would be technically feasible?

Cheers!

Wolf

**Comment by zmk5** [comment]
Yeah, I'm cool with that. Let me know when you want to do it. I think it will be technically feasible because I only really modified the Python files. I didn't touch any of the JS parts yet since I'm still trying to figure out the connections made on some of my free time.
